src/Tickets/Admin/Provider.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * The main service provider for the Tickets Admin area.
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Admin
+ */
+
+namespace TEC\Tickets\Admin;
+
+use tad_DI52_ServiceProvider;
+
+/**
+ * Service provider for the Tickets Admin area.
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Admin
+ */
+class Provider extends tad_DI52_ServiceProvider {
```

Comment:
```suggestion

/**
 * Service provider for the Tickets Admin area.
 *
 * @since   TBD
 * @package TEC\Tickets\Admin
 */
class Provider extends \tad_DI52_ServiceProvider {
```